src/Hosting/Hosting/src/Workers/CronWorkerOption.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace ClickView.Extensions.Hosting.Workers;
+
+using Exceptions;
+
+public record CronWorkerOption
+{
+    public bool AllowJitter { get; }
+    /// <summary>
+    /// Minimum extra time in second delayed before the scheduled task can start
+    /// </summary>
+    public uint MinJitter { get; }
+    /// <summary>
+    /// Maximum extra time in second delayed before the scheduled task can start
+    /// </summary>
+    public uint MaxJitter { get; } = 120;
```

Comment:
I would use `TimeSpan` here as its a bit more clear what the values should be

Reply:
done.